consensus/config/src/committee.rs

```diff
@@ -1,15 +1,13 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-use fastcrypto::traits::KeyPair;
-use std::fmt::{Display, Formatter};
+use crate::{NetworkKeyPair, NetworkPublicKey, ProtocolKeyPair, ProtocolPublicKey};
```

Comment:
Can we change the auto formatter to use types from std crates first, followed by types from external crates, then internal types?